app/src/main/java/com/cornellappdev/transit/ui/screens/HomeScreen.kt

```diff
@@ -125,10 +128,7 @@ fun HomeScreen(
     // Search bar flow
     val searchBarValue = homeViewModel.searchQuery.collectAsState().value
 
-    // Collect flow of rides through API
-    val stopsApiResponse = homeViewModel.stopFlow.collectAsState().value
-    val queryResponse = homeViewModel.queryFlow.collectAsState().value
-    val placesResponse = homeViewModel.placeData.collectAsState().value
+    val placeQueryResponse = homeViewModel.placeQueryFlow.collectAsState().value
```

Comment:
For the future, I'll have any files that have a ton of flows use some internal `ViewState` class to package all the flow values together with `.copy()` calls. It's better practice. But that's for a later PR i suppose.

Reply:
Noted